src/fsipy/automatedPreprocessing/automated_preprocessing.py

```diff
@@ -123,6 +123,7 @@ def run_pre_processing(input_model, verbose_print, smoothing_method, smoothing_f
     # Scale surface
     if scale_factor is not None:
         surface = scale_surface(surface, scale_factor)
+        resampling_step *= scale_factor
```

Comment:
I think it would be good to add a note about this for the `--resampling-step` command line argument. For example, something like this:
``` python
parser.add_argument('-rs', '--resampling-step',
                    default=0.1,
                    type=float,
                    help="Resampling step used to resample centerline in [m]. "
                         "Note: If --scale-factor is used, this step will be adjusted accordingly.")
```
Also, I think we should scale the solid thickness similarly when using `--scale-factor`. We should handle both `constant` and `variable` solid thickness.

Reply:
Agreed! I added note for resampling step as well as scaling of the solid thickness.

Reply:
Thanks! I think that only the `offset` and `min` and `max` distances should be scaled by `scale_factor`, not the `scale` value. I purpose the following changes, but feel free to use what you want:
``` patch
diff --git a/src/fsipy/automatedPreprocessing/automated_preprocessing.py b/src/fsipy/automatedPreprocessing/automated_preprocessing.py
index 531a21e..0ca3158 100755
--- a/src/fsipy/automatedPreprocessing/automated_preprocessing.py
+++ b/src/fsipy/automatedPreprocessing/automated_preprocessing.py
@@ -125,7 +125,6 @@ def run_pre_processing(input_model, verbose_print, smoothing_method, smoothing_f
         print(f"--- Scale model by factor {scale_factor}\n")
         surface = scale_surface(surface, scale_factor)
         resampling_step *= scale_factor
-        solid_thickness_parameters = [scale_factor * i for i in solid_thickness_parameters]
 
     # Check if surface is closed and uncapps model if True
     is_capped = check_if_closed_surface(surface)
@@ -389,11 +388,15 @@ def run_pre_processing(input_model, verbose_print, smoothing_method, smoothing_f
     if solid_thickness == 'variable':
         if not path.isfile(file_name_distance_to_sphere_solid_thickness):
             if len(solid_thickness_parameters) == 4:
-                distance_offset, distance_scale, min_distance, max_distance = solid_thickness_parameters
+                # Apply scale factor to offset, min distance, and max distance
+                if scale_factor is not None:
+                    solid_thickness_parameters[0] *= scale_factor  # Offset
+                    solid_thickness_parameters[2] *= scale_factor  # Min distance
+                    solid_thickness_parameters[3] *= scale_factor  # Max distance
+
                 distance_to_sphere = distance_to_spheres_solid_thickness(distance_to_sphere,
                                                                          file_name_distance_to_sphere_solid_thickness,
-                                                                         distance_offset, distance_scale,
-                                                                         min_distance, max_distance)
+                                                                         *solid_thickness_parameters)
             elif len(solid_thickness_parameters) == 0:
                 distance_to_sphere = distance_to_spheres_solid_thickness(distance_to_sphere,
                                                                          file_name_distance_to_sphere_solid_thickness)
@@ -408,6 +411,10 @@ def run_pre_processing(input_model, verbose_print, smoothing_method, smoothing_f
             print("ERROR: Invalid parameter for constant solid thickness. This should be a " +
                   "single number greater than zero.")
             sys.exit(-1)
+        else:
+            # Apply scale factor to the constant thickness value
+            if scale_factor is not None:
+                solid_thickness_parameters[0] *= scale_factor
 
     # Compute mesh
     if not path.isfile(file_name_vtu_mesh):
@@ -647,13 +654,14 @@ def read_command_line(input_path=None):
     parser.add_argument("-sch5", "--scale-factor-h5",
                         default=1.0,
                         type=float,
-                        help="Scaling factor for HDF5 mesh. Used to scale model to [mm]." +
-                             "Note that probes and other parameters are not scaled.")
+                        help="Scaling factor for HDF5 mesh. Used to scale model to [mm]. " +
+                             "Note that probes and other parameters are not scaled. " +
+                             "Do not use in combination with --scale-factor.")
 
     parser.add_argument('-rs', '--resampling-step',
                         default=0.1,
                         type=float,
-                        help="Resampling step used to resample centerline in [m]." +
+                        help="Resampling step used to resample centerline in [m]. " +
                              "Note: If --scale-factor is used, this step will be adjusted accordingly.")
 
     parser.add_argument('-mp', '--meshing-parameters',
@@ -682,11 +690,13 @@ def read_command_line(input_path=None):
                         type=float,
                         nargs="+",
                         default=[0.3],
-                        help="Parameters for solid thickness [m]. For constant solid thickness, this should be " +
-                             "given as a single float. For 'variable' solid thickness, this should be given as " +
-                             "four floats for the distancetosphere scaling function: 'offset', 'scale', 'min' " +
-                             "and 'max'. For example --solid-thickness-parameters 0 0.1 0.25 0.3 " +
-                             "Note: If --scale-factor is used, solid thickness will be adjusted accordingly.")
+                        help="Parameters for solid thickness [m]. For 'constant' solid thickness, provide a single " +
+                             "float. For 'variable' solid thickness, provide four floats for the distancetosphere " +
+                             "scaling function: 'offset', 'scale', 'min' and 'max'. " +
+                             "For example --solid-thickness-parameters 0 0.1 0.25 0.3. " +
+                             "Note: If --scale-factor is used, 'offset', 'min', and 'max' parameters will be " +
+                             "adjusted accordingly for 'variable' solid thickness, and the constant value will also " +
+                             "be scaled for 'constant' solid thickness.")
 
     parser.add_argument('-mf', '--mesh-format',
                         type=str,

```